2022-04-28 16:57:46	D:\python\www\data_service_monitor\init\init_case.py:18	[error]:pack case error	[err_msg]:'gbk' codec can't decode byte 0xad in position 288: illegal multibyte sequence	
2022-04-28 16:58:26	D:\python\www\data_service_monitor\init\init_case.py:20	[error]:pack case error	[err_msg]:'module' object is not iterable	
2022-12-27 16:45:25	D:\python\www\data_service_monitor\init\init_case.py:20	[error]:pack case error	[err_msg]:'module' object is not iterable	
